plugins/modules/win_regmerge.py

```diff
@@ -21,8 +21,11 @@
     description:
       - The full path including file name to the registry file on the remote machine to be merged
     type: path
-    required: yes
-  compare_key:
+  content:
+    description:
+      - When used instead of O(path), merges the value specified into the Windows registry
+    type: str
```

Comment:
We need to document when the new parameter was added.

```suggestion
    type: str
    version_added: 2.2.0
```